spark3.0/src/main/java/com/splicemachine/sample/KafkaTopicProducer.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2012 - 2020 Splice Machine, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.splicemachine.sample;
+
+import java.sql.Timestamp;
+import java.util.*;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+public class KafkaTopicProducer {
+
+    private String server = null;
+    private long totalEvents = 1;
+    private String topic = null;
+    private KafkaProducer<String, String> producer = null;
+
+    /**
+     * Adds records to a Kafka queue
+     *
+     * @param args args[0] - Kafka Broker URL
+     *             args[1] - Kafka Topic Name
+     *             args[2] - Number of messages to add to the topic
+     *             args[3] - Number of threads, each thread will send the number
+     *                        of messages given in args[2]
+     * @throws Exception
+     */
+    public static void main(String[] args) throws Exception {
+        KafkaTopicProducer kp = new KafkaTopicProducer();
+        kp.server = args[0];
+        kp.topic = args[1];
+        kp.totalEvents = Long.parseLong(args[2]);
+	kp.run(Long.parseLong(args[3]));
```

Comment:
nit: indent

Reply:
Changed in commit https://github.com/splicemachine/splice-machine-spark-connector/pull/31/commits/79eb591fe611c3e080cf221d27d5edb015b759ec